src/providers/wms/qgswmscapabilities.cpp

```diff
@@ -783,6 +783,34 @@ void QgsWmsCapabilities::parseLegendUrl( QDomElement const &e, QgsWmsLegendUrlPr
   QgsDebugMsg( QStringLiteral( "exiting." ) );
 }
 
+void QgsWmsCapabilities::parseMetadataUrl( QDomElement const &e, QgsWmsMetadataUrlProperty &metadataUrlProperty )
+{
+
+  QDomNode n1 = e.firstChild();
+  while ( !n1.isNull() )
+  {
+    QDomElement e1 = n1.toElement(); // try to convert the node to an element.
+    if ( !e1.isNull() )
+    {
+      QString tagName = e1.tagName();
+      if ( tagName.startsWith( QLatin1String( "wms:" ) ) )
+        tagName = tagName.mid( 4 );
+
+      if ( tagName == QLatin1String( "Format" ) )
```

Comment:
I'm not familiar with the specs, but should these tests be case insensitive?

```suggestion
      if ( tagName.compare( QLatin1String( "Format" ), Qt::CaseInsensitive ) == 0 )
```

Reply:
Well the specs says that the format tag will be "Format" for WMS 1.3.0.

Reply:
I think using a case insensitive compare won't harm anything and will improve compatibility with systems that are not 100% compliant.

Reply:
@nyalldawson @timlinux got you.

Reply:
Resolved here https://github.com/qgis/QGIS/pull/33467/commits/e8c3a8c987eb682f7bb92849889e8e53905d4123